packages/block-library/src/group/editor.scss

```diff
@@ -22,3 +22,16 @@
 		margin-bottom: $block-padding + $grid-unit-05;
 	}
 }
+
+
+.block-library-group-block-html-element-control {
```

Comment:
These can be removed in combination with the wrapper removal.

Reply:
Done